tests/testqPrefDisplay.cpp

```diff
@@ -40,13 +41,13 @@ void TestQPrefDisplay::test_set_struct()
 	auto display = qPrefDisplay::instance();
 
 	display->set_display_invalid_dives(true);
-	display->set_divelist_font("comic");
+	display->set_divelist_font("doNotCareAtAll");
```

Comment:
Nice - of course now the commit message doesn't match anymore... :-)
I'm happy to adjust that as I apply the patches - I'm just waiting for Travis to finish